extensions/wikia/ImageReview/modules/PromoteImage/PromoteImageReviewHelper.class.php

```diff
@@ -117,15 +117,15 @@ protected function saveStats($statsInsert, $sqlWhere, $action) {
 				case '':
 					//	$stats['reviewer'] += count($images);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)